doc/changes.rst

```diff
@@ -5,7 +5,9 @@ Change Log
 1.9.3 (unreleased)
 ------------------
 
-* No changes yet.
+* Added :func:`desiutil.plots.plot_sky` to make all-sky plots (`PR #52`_).
```

Comment:
It looks like a lot more work was done than what this sentence implies.  Also, in all the rewriting, does that function still exit, or apply?